src/components/AuthInput.tsx

```diff
@@ -1,53 +1,61 @@
-import { useState } from "react";
+import { useState, type FocusEvent, type FocusEventHandler, type InputHTMLAttributes } from "react";
 import Input from "./Input";
 import icEyeVisible from "../assets/ic_eye_visible.svg";
 import icEyeInvisible from "../assets/ic_eye_invisible.svg";
 import "./AuthInput.css";
 
-function PasswordInput ({ name, className, onFocusout, valid, wrongMessage, ...props }) {
+interface PasswordProps extends InputHTMLAttributes<HTMLInputElement> {
+    name: string;
+    label: string;
+    className?: string; 
+    onFocusout?: FocusEventHandler; 
+    valid?: string; 
+    wrongMessage: string;
```

Comment:
💊 제안
옵셔널값들이 뒤에 오게 해주시는 것이 가독성 측면에서 더 좋습니다.

```suggestion
    name: string;
    label: string;
    wrongMessage: string;
    className?: string; 
    onFocusout?: FocusEventHandler; 
    valid?: string; 
```